src/proto/event.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package virtual_people;
+
+// Placeholder for DemoBucket message.
+message DemoBucket {}
+
+// Placeholder for LabelerInput message.
+message LabelerInput {}
+
+// Placeholder for VirtualPersonActivity message.
+message VirtualPersonActivity {}
```

Comment:
`virtual person` vs `virtual people`, have we finalized the name yet?

Reply:
"virtual person" represents a single "virtual person", while "virtual people" refers to a set of "virtual person"s.

Reply:
Might be a useful piece of information to add somewhere, in a README or in the proto files themselves.

Reply:
The explanation was added in the comments of virtual_person_activities in model.proto.